.github/workflows/opentelemetry.yml

```diff
@@ -7,6 +7,7 @@ on:
       - Auto Assign to Project(s)
       - Issue Labeler
       - ci
+      - ci-docs
```

Comment:
Unrequired, it uses the same name

Reply:
https://github.com/elastic/apm-agent-ruby/pull/1378/commits/f446791576de77fa0824da2b49bf930dfdced355